syntax = "proto3";

package calculator;

// The  service definition.
service Calculator {
  // Defines the Remote Procedure Calls available on the Service
  rpc SumNumbers(CalcRequest) returns (NumericResult);
  rpc SubtractNumbers(CalcRequest) returns (NumericResult);
}

message CalcRequest {
  int32 numberA = 1;
  int32 numberB = 2;
}

message NumericResult {
  int32 sum = 1;
}
